source/Octopus.Client.E2ETests/Octopus.Client.E2ETests.csproj

```diff
@@ -10,32 +10,28 @@
         <LangVersion>latest</LangVersion>
     </PropertyGroup>
     <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
-        <TargetFrameworks>net462;net48;net6.0;net8.0</TargetFrameworks>
+        <TargetFrameworks>net462;net48;net8.0</TargetFrameworks>
     </PropertyGroup>
     <PropertyGroup Condition="$([MSBuild]::IsOSUnixLike())">
-        <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+        <TargetFramework>net8.0</TargetFramework>
     </PropertyGroup>
     <ItemGroup>
         <ProjectReference Include="..\Octopus.Client\Octopus.Client.csproj" />
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.DotNet.Analyzers.Compatibility" Version="0.2.12-alpha">
```

Comment:
`Microsoft.DotNet.Analyzers.Compatibility` has been deprecated forever, the equivalent functionality is part of the .NET SDK since .NET 5